docs/product/components/textarea.html

```diff
@@ -8,13 +8,13 @@
     {% header "h2", "Base style" %}
     <div class="stacks-preview">
 {% highlight html %}
-<div class="d-flex fd-column gs4 gsy">
+<div class="d-flex fd-column gx4">
```

Comment:
## Change requested

Since `<label>` elements are rendered as `display: table`, the gap property doesn't apply. Any `<label>` with parents that include a gap property need either to have a margin added or have their `display` property altered to honor `gap`.

Reply:
Also for `textarea` cannot see visual differences.

Reply:
Ah, this is the same issue as reference in https://github.com/StackExchange/Stacks/pull/1157#discussion_r1004616938

> Ah, I was a little hasty in assuming the `<legend>` issue was here. Backstop show some significant differences on this page and it looks to be due to `.s-input-message` being rendered with the `<p>` tag, which comes with margins that were previously overwritten with the `.gs4` class.

Reply:
👍